src/ekf.cpp

```diff
@@ -362,7 +362,26 @@ namespace RobotLocalization
     estimateErrorCovariance_ = (transferFunctionJacobian_ *
                                 estimateErrorCovariance_ *
                                 transferFunctionJacobian_.transpose());
-    estimateErrorCovariance_.noalias() += (processNoiseCovariance_ * delta);
+
+    // Compute processNoiseCovariance scaling if nessesary
+    Eigen::MatrixXd processNoiseCovariance = processNoiseCovariance_;
+    if (processNoiseCovarianceScaler_ > 0)
```

Comment:
Can we change the set method such that it sets a bool and the value, and then we check the bool here? Double comparisons to 0 make me nervous.